src/parse.rs

```diff
@@ -150,9 +154,9 @@ where
                     end: location,
                 },
             )
-        } else if statements.is_err() {
+        } else if parse_result.is_err() {
             // Then parse errors
-            statements
+            parse_result
         } else if let Some((start, _, end)) = self.next_tok() {
             // there are still more tokens
             let expected = vec!["An import, const, type, or function.".to_string()];
```

Comment:
This is the error message I meant- this will be incorrect if there is an end-of-file during a non-statement sequence, right?

Reply:
oh whoops sorry, it's probably fine for now, `parse_expression_sequence` is only available in test and we might want to change things when making that public for some reason.

Reply:
Ah cool! Ta